src/main/java/io/github/kawamuray/wasmtime/NativeLibraryLoader.java

```diff
@@ -96,6 +97,9 @@ private static Platform detectPlatform() {
         if (os.contains("mac os") || os.contains("darwin")) {
             return Platform.MACOS;
         }
+        if(os.contains("windows") || os.contains("Windows")){
```

Comment:
style nit: please align code style with others :) 

`if" "(os.contains("windows") || os.contains("Windows"))" "{`

Reply:
OK. Thank you.